cmd/experimental/kjobctl/pkg/builder/builder_test.go

```diff
@@ -448,7 +448,12 @@ func TestBuilder(t *testing.T) {
 				return
 			}
 
-			if diff := cmp.Diff(tc.wantObj, gotObjs); diff != "" {
+			var gotObjs []runtime.Object
+			if rootObj != nil {
+				gotObjs = append(gotObjs, rootObj)
+			}
+			gotObjs = append(gotObjs, childObjs...)
```

Comment:
nit:
```suggestion
			gotObjs := append([]runtime.Object{rootObj}, childObjs...)
```

Is there any scenario in which the root object is nil?

(the same goes for some other tests)

Reply:
Yes, when we testing with errors.

Reply:
> Yes, when we test it with errors.

... :)

then maybe is better to use `wantRootObject` and `wantChildObjects`

Reply:
Done